### ISSUE TITLE
After version 1.96 some 3dtiles display is not normal

### ISSUE THREAD

**Issue by hemincan** [CesiumGS/cesium#11115]
Sandcastle example:[this is example](https://sandcastle.cesium.com/#c=bVJdb5swFP0rFk9Eak3ollTdaLSIdIw0kKXQsES8EOMEJ2BT24TAtP8+0zRaN+3Juj4f9/pcGwZweEIlsLEgVTENQIIQFgJIBhpWcUAYBYkQWIqYnjnQZRSmeJtUuRy/kkN2wBTcg1jDzTTbOIjMydR9bl3TJ65w6dMA2e7QPZQ/lvb0DirSS+ocFMltVtG6WBUPjReYxG/H7SxcDObRs1ztH8y1bWZ+ezh5zmrgO4vTOvLIzJ6Wa2Xm7Xe1H9Qkib723T07+aHX91v3o7/36u0CDr81k+g6fHysl851ujvug9nd+HuZoeXW5OyGyCYbRrdZEE12sfY5pohRIcGR4Bpz9RCK67dA4PL1To819FrbjMqEUMxjrad0F6UkOVYZKenZAwqEKYYlJwWR5IgFTNJUjyl4b30+PkzCs1j/2eEAVDz/BP5E/YSF2gPCcMtZMe424aa6ORjemLf93lUn+dWLaTfMpXfLWBEy/W0mBWpXmiVkk+NRR/9CipJx2fXRITQkLso8kVgYmwodsIRIiM7NMi4SKyVHQNL7/2QAUK4+h0K2VZ4HpMWxNrIMxf9LlrMkJXQ3P2KeJ42idGNYmTmanQEIoWWosmv6r1Yylm8S/s73Nw)
![image](https://user-images.githubusercontent.com/32860032/221484236-9308dd55-3f91-4e1e-8351-a03ba9a30567.png)

![image](https://user-images.githubusercontent.com/32860032/221484617-a6639580-c69a-49a5-9854-ab1961f71450.png)

But it is displayed normally in 1.96 and below.
![image](https://user-images.githubusercontent.com/32860032/221484431-7d1d00fa-73f1-4c28-be5b-defb90d314e7.png)

**Comment by ggetz** [comment]
Thanks for the report @hemincan. I can confirm this is a bug after we [overhauled glTF loading](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md#197---2022-09-01).

**Comment by jinjun1994** [comment]
same problem

**Comment by jinjun1994** [comment]
How to fix this bug? I am in a hurry. @ggetz

**Comment by jinjun1994** [comment]
I want to fix this bug, could you give me some clues as I cannot locate it? Thank you very much! @ggetz

**Comment by ggetz** [comment]
Hi @jinjun1994, this would likely need further debugging to determine the exact issue. I would begin looking in `GltfLoader` which loads the glTF embedded in the tile, and creates the resources needed to render the glTF. Then `ModelSceneGraph` prepares the resources for rendering.

**Comment by jinjun1994** [comment]
@ggetz  
![image](https://user-images.githubusercontent.com/21867818/224362950-bb013528-0d3e-4caa-ba31-62d057e993be.png)

The issue may be related to the size of the index buffer？ Under the new glTF loader architecture, the glTF file is not displayed. The command is "drawElements: TRIANGLES, 269259, UNSIGNED_SHORT, 0". However, when changed to "UNSIGNED_SHORT", although the object is deformed, it is displayed.

**Comment by dgjkl** [comment]
Hi @jinjun1994 ，Have you found a solution yet？Cesium version is 1.102

**Comment by dgjkl** [comment]
This issue is related to the index buffer.This could be the reason：
GltfLoader Line 1253 :  indices.count = accessor.count;
this count is not equal to indexBufferLoader.buffer.numberOfIndices;

**Comment by MSNjojo** [comment]
> This issue is related to the index buffer.This could be the reason： GltfLoader Line 1253 : indices.count = accessor.count; this count is not equal to indexBufferLoader.buffer.numberOfIndices;

yes, the temporary work around is to set count = defaultValue(count, indexBuffer.numberOfIndices);

**Comment by MSNjojo** [comment]
> This issue is related to the index buffer.This could be the reason： GltfLoader Line 1253 : indices.count = accessor.count; this count is not equal to indexBufferLoader.buffer.numberOfIndices;

yes, here is my work around

**Comment by MSNjojo** [comment]
> This issue is related to the index buffer.This could be the reason： GltfLoader Line 1253 : indices.count = accessor.count; this count is not equal to indexBufferLoader.buffer.numberOfIndices;

yes, here is my work around
context.js
count = defaultValue(count, indexBuffer.numberOfIndices); => count = indexBuffer.numberOfIndices;

I guess there are some errors in  model loader or the process of parsing data. It results in numberOfIndices  not  equal to count
